lib/paper_trail/has_paper_trail.rb

```diff
@@ -269,6 +270,7 @@ def touch_with_version(name = nil)
 
         attributes.each { |column| write_attribute(column, current_time) }
         save!
+        record_update(true) if self.class.paper_trail_options[:on] == []
```

Comment:
In my code I need to do touch_with_version because something in my metadata on the version has changed, not the object itself, but this line stops me from actually doing a record_update since my model has option[:on] set to nil, which will default it to [:create, :update, :destroy]. So is tocuh_with_version only meant to be used on models without automatic versions on update/create/destroy?